.claude/skills/dimension-fact-identification/SKILL.md

```
@@ -0,0 +1,365 @@
+
+Claude:
+  → Major チェック (未マーク Dim、Fact grain 未文書化) を優先実行
+  → 該当があれば即対応案を提示、無ければ Conformed Dim 候補を見せる
+```
+
+## 可視化との連携
+
+`ontology-visualize` Skill と組み合わせると、Dim-Fact グラフが見える:
+
+> 「全 Fact と Dim を Mermaid で見せて」 → ontology-visualize Skill 起動
+>   (内部で `?fact rdfs:domain|rdfs:range ?dim` の CONSTRUCT)
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)